ParabolInc/parabol
# Make sure that users can always see agenda items in check-in

Fixes #5343 

Previously, if the height of the meeting members exceeded the available space in the sidebar, the user couldn't see the agenda items:

![old-view](https://user-images.githubusercontent.com/39854876/132241251-1502b1d3-2bca-485b-ad30-842d54bb543a.png)

This PR ensures that 5 agenda items are always visible (if there are at least 5 agenda items):
![new-view](https://user-images.githubusercontent.com/39854876/132241502-9c9b915c-16ef-4c84-9b2d-9487f4f7f9a7.png)

### To Test:

- create a check-in meeting
- add a few other users to the meeting
- reduce the height of the page using the dev tools so that the Icebreaker / Solo Updates section takes up most of the sidebar height
- note that regardless of the number of meeting members in the Icebreaker / Solo Updates phase, you can still see 5 agenda items (if there are at least 5)

Note: there are some typecheck errors in files that this PR doesn't touch. These should be fixed by #5356
